app/components/Sidebar/components/CollectionLink.tsx

```diff
@@ -242,7 +255,7 @@ function CollectionLink({
                 />
               }
               exact={false}
-              depth={0.5}
+              depth={0.6}
```

Comment:
keep this at `0.5`. It's a little tight on the display but it's not too bad because it only shows on hover